cli/dts/lib.deno.unstable.d.ts

```diff
@@ -1181,6 +1181,35 @@ declare namespace Deno {
    * then the underlying HttpConn resource is closed automatically.
    */
   export function serveHttp(conn: Conn): HttpConn;
+
+  /** **UNSTABLE**: New option, permissions are yet to be vetted.
+   *
+   * Use the peermissions options to change the level of access the test will
+   * have. By default it will inherit the permissions of its parent thread. The permissions
+   * of a test can't be extended beyond its parent's permissions reach.
+   * - "inherit" will take the permissions of the thread the test is created in
```

Comment:
This explanation is not at all clear - is it applicable to the the top level `permissions` property, or is it about specific permission flags e.g. `env: true`.

Maybe add examples instead.

Reply:
Yeah lazily copied from the worker permissions section.